docs/src/rules/logical-assignment-operators.md

```diff
@@ -0,0 +1,122 @@
+---
+title: logical-assignment-operators
+layout: doc
+edit_link: https://github.com/eslint/eslint/edit/main/docs/src/rules/logical-assignment-operators.md
+rule_type: suggestion
+---
+
+
+
+Require or disallow assignment logical operator shorthand
+
+## Rule Details
+
+This rule requires or disallows logical assignment operator shorthand.  
+ES2021 introduces the assignment operator shorthand for the logical operators `||`, `&&` and `??`.
+Before this was only allowed for mathematical operations such as `+` or `*` (see the rule [operator-assignment](./operator-assignment)).
+The shorthand can be used if the assignment target and the left expression of a logical expression are the same.
+For example `a = a || b` can be shortened to `a ||= b`.
+
+### Options
+
+This rule has a string and an object option.
+String option:
+
+* `"always"` (default)
+* `"never"`
+
+Object option (only available if string option is set to `"always"`):
+
+* `"enforceForIfStatements": false`(default) Do *not* check for equivalent if statements
+
+* `"enforceForIfStatements": true` Check for equivalent if statements
```

Comment:
```suggestion
* `"enforceForIfStatements": false` (default) Do *not* check for equivalent `if` statements
* `"enforceForIfStatements": true` Check for equivalent `if` statements
```

There's an extra line between list items, and it would be nice to add backticks around `if`.